x-pack/plugins/ml/public/application/services/ml_api_service/results.ts

```diff
@@ -114,4 +116,19 @@ export const resultsApiProvider = (httpService: HttpService) => ({
       body,
     });
   },
+
+  getStoppedPartitions(
+    jobIds: string[],
+    fieldToBucket?: typeof JOB_ID | typeof PARTITION_FIELD_VALUE
+  ) {
+    const body = JSON.stringify({
+      jobIds,
+      fieldToBucket,
+    });
+    return httpService.http<any>({
```

Comment:
looks like this returns a `GetStoppedPartitionResult` 
That interface be moved to a common location and used server side and here.

Reply:
Updated in https://github.com/elastic/kibana/pull/74592/commits/a94ecaf246b7f84168eb296b5c2c1a75471353cc